typescript/src/schema/proto.ts

```diff
@@ -575,29 +570,15 @@ export class Proto {
   constructor(
     fd: plugin.google.protobuf.IFileDescriptorProto,
     packageName: string,
+    allMessages: MessagesMap,
     allResourceDatabase: ResourceDatabase,
     resourceDatabase: ResourceDatabase,
     options: Options
   ) {
-    fd.enumType = fd.enumType || [];
-    fd.messageType = fd.messageType || [];
```

Comment:
curious that we don't wanna handle enum type any more?

Reply:
yeah, I figured, we never use enums in the generator. Less code!

Reply:
sounds great! thank you!